docs/src/styles/index.css

```diff
@@ -1,3 +1,13 @@
+/* Docs styles */
```

Comment:
This is the new single CSS file that the docs will load globally with the proper order of imports.